File: sample/src/main/java/contacts/sample/cheatsheet/blockednumbers/java/InsertBlockedNumbersActivity.java
package contacts.sample.cheatsheet.blockednumbers.java;

import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.blockednumbers.BlockedNumbersInsert;
import contacts.core.entities.NewBlockedNumber;

public class InsertBlockedNumbersActivity extends Activity {

    BlockedNumbersInsert.Result insertBlockedNumber() {
        return ContactsFactory.create(this)
                .blockedNumbers()
                .insert()
                .blockedNumbers(new NewBlockedNumber("555-555-5555"))
                .commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/blockednumbers/kotlin/InsertBlockedNumbersActivity.kt
package contacts.sample.cheatsheet.blockednumbers.kotlin

import android.app.Activity
import contacts.core.Contacts
import contacts.core.blockednumbers.BlockedNumbersInsert

class InsertBlockedNumbersActivity : Activity() {

    fun insertBlockedNumber(): BlockedNumbersInsert.Result = Contacts(this)
        .blockedNumbers()
        .insert()
        .blockedNumber {
            number = "555-555-5555"
        }
        .commit()
}